fiftyone/server/view.py

```diff
@@ -698,21 +701,22 @@ def _get_filtered_path(view, path, filtered_fields, label_tags):
     return "___%s" % path
 
 
-def _add_frame_labels_tags(path, view):
+def _add_frame_labels_tags(path, field, view):
     path = path[len("frames.") :]
+    items = "%s.%s" % (path, field.document_type._LABEL_LIST_FIELD)
```

Comment:
this is breaking when I load a specific dataset
```
ERROR:root:'ListField' object has no attribute 'document_type'
Traceback (most recent call last):
  File "/Users/maniaskari/code/fiftyone/fiftyone/server/decorators.py", line 34, in wrapper
    response = await func(endpoint, request, data, *args)
  File "/Users/maniaskari/code/fiftyone/fiftyone/server/routes/samples.py", line 29, in post
    results = await paginate_samples(
  File "/Users/maniaskari/code/fiftyone/fiftyone/server/samples.py", line 75, in paginate_samples
    view = fosv.get_view(
  File "/Users/maniaskari/code/fiftyone/fiftyone/server/view.py", line 126, in get_view
    view = get_extended_view(
  File "/Users/maniaskari/code/fiftyone/fiftyone/server/view.py", line 221, in get_extended_view
    view = _add_labels_tags_counts(view, filtered_labels, label_tags)
  File "/Users/maniaskari/code/fiftyone/fiftyone/server/view.py", line 275, in _add_labels_tags_counts
    view = add_tags(path, field, view)
  File "/Users/maniaskari/code/fiftyone/fiftyone/server/view.py", line 735, in _add_labels_tags
    items = "%s.%s" % (path, field.document_type._LABEL_LIST_FIELD)
AttributeError: 'ListField' object has no attribute 'document_type'
```

Reply:
Will fix, but there are a plethora of issues with raw label lists which looks like the field type here